R/timePerSubject.R

```diff
@@ -0,0 +1,57 @@
+#' \linkS4class{SummarizedExperiment} class object stores time information about
+#' each sample in `colData` field. Time information can be categorized by
+#' per subject(individual) and then used with the selected time operation.
+#' The resulting vector is added as a new column of the`colData` field.
+#'
+#' @param se : \linkS4class{SummarizedExperiment} class object
+#' @param time_field : column vector containing the time information
+#' @param subject_field : column vector containing the subject(individual)
+#' information
+#' @param operation : a function applied to `time_field`
+#' @param new_field : character class object refers to the name of a
+#' newly added field
+#'
+#' @examples
+#' library(miaTime)
+#' data(hitchip1006)
+#' se <- hitchip1006
+#'
+#' hitchipTime <- timePerSubject(se, time_field = "time",
+#'     subject_field = "subject", operation ="rev", new_field = "reversed")
```

Comment:
also added `decreasing = TRUE ` to end and didnt' work.  `error: x unused argument (decreasing = TRUE)`